src/controllers/controlpickermenu.cpp

```diff
@@ -890,18 +1098,37 @@ void ControlPickerMenu::addPlayerControl(QString control, QString controlTitle,
         }
     }
 
-    for (int i = 1; samplerControls && i <= iNumSamplers; ++i) {
-        // PlayerManager::groupForSampler is 0-indexed.
-        QString prefix = m_samplerStr.arg(i);
-        QString group = PlayerManager::groupForSampler(i - 1);
-        addSingleControl(group, control, controlTitle, controlDescription,
-                         controlMenu, prefix, prefix);
-
+    if (samplerControls) {
+        parented_ptr<QMenu> samplerControlMenu = make_parented<QMenu>(tr("Samplers"), controlMenu);
+        controlMenu->addMenu(samplerControlMenu);
+        parented_ptr<QMenu> samplerResetControlMenu = nullptr;
```

Comment:
Does parented_ptr make sense if the variable is not stored?

Reply:
probabyl not. I'll check.
I confess I just lazily copied that from the main menu above.